_layouts/media.html

```diff
@@ -105,6 +105,7 @@
 
 var talks = [
 	{ id: "wSpPjUPIEVU", title: "OSDC 2015 Foreman in your data center", author: "Stephen Benjamin" },
+	{ id: "b18Us1nAKMA", title: "PuppetCamp 2015: Selling management and developers on Puppet", author: "David Patterson"},
```

Comment:
No worries, I think if you're in a playlist YouTube doesn't update the top URL.  I think you can edit it in GitHub's web interface too (on the Files changed tab),  but @eLobato's amend suggestion is the best way to do it so it stays just one commit.